src/index.ts

```diff
@@ -63,6 +63,10 @@ export class GCEImages {
     this.OS_URLS = GCEImages.OS_URLS;
   }
 
+  async getProjectId() {
```

Comment:
This feels a bit out of place, but let me know if I missed a user report or something similar. If the user is getting the latest images from their project, I believe they will know the project ID, and I don't think it will need to adapt to the environment, as it is just in the one fixed location.

Reply:
ahhhh - I should do this in another PR.  This is a method that's available in all of our GAPIC generated libraries, so I added it here for consistency.  @alexander-fenster thoughts?

Reply:
Since this is not a GAPIC library I don't think we need to have this consistency, but I also see no harm of it here.